tests/Oliver.IntegrationTests/DeployTest.cs

```diff
@@ -0,0 +1,140 @@
+﻿using FluentAssertions;
+using Oliver.Common.Models;
+using Oliver.IntegrationTests.Helpers;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Oliver.IntegrationTests
+{
+    [Collection("Integration")]
+    public class DeployTest : ClientServerTestBase
+    {
+        [Fact]
+        public async Task Deploy_Should_Execute_Properly()
+        {
+            // Arrange
+            var packageId = await CreatePackageAsync();
+            var templateId = await CreateTemplateAsync();
+            var variableSetId = await CreateVariableSetAsync();
+
+            // Action
+            var executionId = await CreateExecutionAsync(templateId, variableSetId);
+            await Task.Delay(5000); // give client some time to work...
+
+            // Assert
+            var execution = await this.api.GetExecutionAsync(executionId);
+            execution.State.Should().Be(Execution.ExecutionState.Successed, "\n" + string.Join('\n',
+                execution.StepsStates
```

Comment:
Could you explain why we concat execution steps with ClientLog? I suppose all required information should be already in ClientLog, shouldn't it?

Reply:
i guess, it is just an insurance for unlogged actions...